src/ol/interaction/dragboxinteraction.js

```diff
@@ -112,11 +112,35 @@ ol.interaction.DragBox = function(opt_options) {
   this.condition_ = options.condition ?
       options.condition : ol.events.condition.always;
 
+  /**
+   * @private
+   * @type {Function}
```

Comment:
The type should be `ol.interaction.DragBoxOptions.BoxEndConditionType`. And I think it could be shortened to `ol.interaction.DragBoxEndConditionType` everywhere.